### DISCUSSION TITLE
Cycle de vie et états REP vs MAJ

### DISCUSSION THREAD

**Discussion by MJMESS** [MTESSDev/FRW#311] [Annonces]
Bonjour, 
lorsque nous reprenons un formulaire déjà créé, nous voyons que l'état est MAJ même si nous n'effectuons pas de changement. 

Devons nous considérons les états MAJ et REP comme étant un formulaire en cours ?

Nous aimerions connaître dans quel circonstance le formulaire devient à l'état "REP - Repris par l'utilisateur" ? 

 Merci à l'avance!

**Comment by Hutson48** [comment]
Bonjour,

L'état "REP - Repris par l'utilisateur" est appliqué lorsque l'utilisateur passe par la page de reprise. Cependant, en mode authentifié, cette page est invisible, ce qui fait que l'état "REP" est transitoire. Il est appliqué peu importe la situation :

- Formulaire fraîchement créé,
- Formulaire repris après plusieurs jours
- Formulaire repris via un lien envoyé par courriel (mode anonyme).

Une fois que l'utilisateur commence à naviguer ou clique dans le formulaire ouvert, l'état passe automatiquement à "MAJ". Ainsi, l'état "REP" a surtout un rôle de traçabilité et ne dure généralement que quelques secondes.

En résumé, vous pouvez considérer que les états "REP" et "MAJ" désignent un formulaire en cours, mais l'état "REP" est temporaire et ne représente pas une étape réelle dans le cycle de vie.

N'hésitez pas à me revenir si vous avez d'autres questions,
